Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Coordinate/GetByStreetcodeId/GetCoordinatesByStreetcodeIdHandler.cs

```diff
@@ -30,10 +33,14 @@ public async Task<Result<IEnumerable<StreetcodeCoordinateDTO>>> Handle(GetCoordi
 
         if (coordinates is null)
         {
-            return Result.Fail(new Error($"Cannot find a coordinates by a streetcode id: {request.StreetcodeId}"));
+            string errorMsg = $"Cannot find a coordinates by a streetcode id: {request.StreetcodeId}";
+            _logger?.LogError("GetCoordinatesByStreetcodeIdQuery handled with an error");
```

Comment:
In what case logger can be null? And what about using string constants?

Reply:
1. Yes you are right. Removed it.
2. About constants I answered Max Bratys above. Here it is impossible because this message is dynamicaly changed. And Valerii is currentrly working on error messages systematization.